roles/aap_setup_install/defaults/main.yml

```diff
@@ -66,7 +69,7 @@ ah_hostname: "{{ (aap_setup_prep_inv_nodes['automationhub'].keys() | list)[0] }}
 # ah_oauthtoken: ""
 # ah_validate_certs: false
 
-eda_hostname: "{{ (aap_setup_prep_inv_nodes['automationedacontroller'].keys() | list)[0] }}"
+eda_hostname: "{{ (aap_setup_prep_inv_nodes[__aap_setup_inst_eda_group_name].keys() | list)[0] }}"
```

Comment:
this is the one part just looking over quickly that seems wrong and I would like to double check because the other components are not like this line.

Reply:
The group name for eda is different between the regular setup installer and the containerized installer. It is one of the inconstancies between the installer inventory.yml files.

Reply:
but then wouldn't this line be different for all of them for container option but not for setup? and if so then wouldn't this be wrong for setup?

Reply:
`__aap_setup_inst_eda_group_name` is set in the role vars: 

```yaml
_aap_setup_inst_eda_group_name: "{{ (aap_setup_inst_containerized | bool) | ternary('automationeda', 'automationedacontroller') }}"
```

If we are not a containerized install then it is set to 'automationedacontroller'

Reply:
Ok but it doesn't look like it's set that way for hub and controller shouldn't it be the same?

Reply:
ah nvm I understand now what you mean